dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/components/ComponentBase.kt

```diff
@@ -32,6 +34,21 @@ open class ComponentBase(
      */
     var explanation: String? = null
 
+    /**
+     * The unit of a string component
+     */
+    var unit: String? = null
```

Comment:
This does not belong in the ComponentBase. It should only be in the StringComponent!

Reply:
Fixed.